hack/generate/changelog/util/fragment_test.go

```diff
@@ -370,3 +370,36 @@ func TestGitPullRequestNumberGetter_parsePRNumber(t *testing.T) {
 		})
 	}
 }
+
+func Test_gitPullRequestNumberGetter_getCommitMessage(t *testing.T) {
+	type args struct {
+		filename string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name:    "should fail when the fragment file cannot be found in any commit",
+			args:    args{filename: "/does/not/exist"},
+			wantErr: true,
+		},
+		{
+			name:    "should work successfully when there is a commit with the fragment file",
+			args:    args{filename: "testdata/valid/fragment1.yaml"},
```

Comment:
Nit: would this test have failed before this change of the similarity index?

Reply:
nope, change of similarity index doesn't have any impact on this test failing/passing (although it would have had some impact on the commit that was returned)